### PR TITLE
form object

### PR DESCRIPTION
**Pull request by brianteeman**
this simple pr for core review only just updates the text in the docblocks. Some of the instance of "a JForm object" had already been updated. I stumbled across this today so made the quick PR

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: administrator/components/com_categories/src/Model/CategoryModel.php [hunk 1]
```diff
@@ -242,7 +242,7 @@ public function getItem($pk = null)
      * @param   array    $data      Data for the form.
      * @param   boolean  $loadData  True if the form is to load its own data (default case), false if not.
      *
-     * @return  Form|boolean  A JForm object on success, false on failure
+     * @return  Form|boolean  a Form object on success, false on failure
```
Comment (heelc29) [inline administrator/components/com_categories/src/Model/CategoryModel.php]:
```suggestion
     * @return  Form|boolean  A Form object on success, false on failure
```
Capitalization?

File: administrator/components/com_config/src/Model/ApplicationModel.php [hunk 2]
```diff
@@ -67,7 +67,7 @@ class ApplicationModel extends FormModel implements MailerFactoryAwareInterface
      * @param   array    $data      Data for the form.
      * @param   boolean  $loadData  True if the form is to load its own data (default case), false if not.
      *
-     * @return  mixed  A JForm object on success, false on failure
+     * @return  mixed  a Form object on success, false on failure
```
Comment (brianteeman) [inline administrator/components/com_config/src/Model/ApplicationModel.php]:
```suggestion
     * @return  mixed  A Form object on success, false on failure
```

**Comment by heelc29** [comment]
I have tested this item :white_check_mark: successfully on b906c4dedd41d658ff2bae2ffb1f12f06dda79ed<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43246">issues.joomla.org/tracker/joomla-cms/43246</a>.</sub>

**Comment by LadySolveig** [comment]
Thank you @brianteeman and also for support @heelc29

**Comment by brianteeman** [comment]
thanks

**Pull request merged**